x/concentrated-liquidity/incentives.go

```diff
@@ -213,3 +218,44 @@ func calcAccruedIncentivesForAccum(ctx sdk.Context, accumUptime time.Duration, q
 
 	return incentivesToAddToCurAccum, poolIncentiveRecords, nil
 }
+
+// nolint: unused
+// setIncentiveRecords sets the passed in incentive records in state
+func (k Keeper) setIncentiveRecord(ctx sdk.Context, incentiveRecord types.IncentiveRecord) {
+	// Potentially need to set the pool's incentiveRecord field as well
```

Comment:
What do you mean by this comment?

Reply:
This was from before I removed `incentiveRecord` as a pool field – deleted this comment now.